src/JSONNestedNode.js

```diff
@@ -106,6 +108,16 @@ export default class JSONNestedNode extends React.Component {
 
   shouldComponentUpdate = shouldPureComponentUpdate;
 
+  isShouldExtract = () => {
```

Comment:
Instead of this, make handleNodeClick and handleMouseOver do nothing and not stopPropagation if their keypath is deeper than maxClickableNodeDepth.